README.md

```diff
@@ -114,6 +114,8 @@ You need to create a bunch of Slack Slash commands, each pointing at the same UR
 /readyforqa [repo]
 /qapass [repo]
 /assigned [repo|*] [login]
+/openpullrequests [repo]
```

Comment:
I believe `/openpullrequests` has an argument that is an integer `number of days since project active` and you cannot filter specifically by a repo.

Maybe something like `/openpullrequests [days]`?